assembl/static2/js/app/components/debate/brightMirror/sideComment/innerBoxView.jsx

```diff
@@ -44,6 +46,9 @@ class InnerBoxView extends React.Component<Props, State> {
       connectedUserCan(Permissions.EDIT_MY_POST) &&
       // Prevent editing reply for now (will be added in another ticket)
       setEditMode;
+    const canDelete =
+      (creator && String(creator.userId) === getConnectedUserId() && connectedUserCan(Permissions.DELETE_MY_POST)) ||
```

Comment:
refactoring: add a new utility isConnectedUser(userId):
`creator && String(creator.userId) === getConnectedUserId() to isConnectedUser(creator && creator.userId)`